mayastor/src/bdev/dev/nvmx/channel.rs

```diff
@@ -191,7 +191,7 @@ extern "C" fn disconnected_qpair_cb(
     qpair: *mut spdk_nvme_qpair,
     _ctx: *mut c_void,
 ) {
-    warn!("NVMe qpair disconnected !");
+    warn!("NVMe qpair disconnected, qpair={:p}", qpair);
```

Comment:
We use [`tracing`](https://github.com/tokio-rs/tracing) so you could use this in a structured manner!

https://github.com/openebs/Mayastor/blob/ce2ca62c841b5715e6c2b482f3d614023025f618/mayastor/Cargo.toml#L75-L79

You can see ["Recording fields"](https://tracing-rs.netlify.app/tracing/index.html#recording-fields), maybe like:

```rust
tracing::warn!(
    ?qpair,
    "NVMe qpair disconnected",
);
```

Reply:
Thanks, fixed.